feature-libs/my-account/organization/src/components/cost-center/form/cost-center-form.service.ts

```diff
@@ -32,4 +40,16 @@ export class CostCenterFormService {
       })
     );
   }
+
+  private codeValidator(
```

Comment:
it seems as will be used not only for cost centers, please move it to other place

Reply:
discussed and decided to remove it